packages/modules-sdk/src/loaders/module-loader.ts

```diff
@@ -1,38 +1,41 @@
-import { asFunction, asValue } from "awilix"
-import { trackInstallation } from "medusa-telemetry"
+import { EOL } from "os"
+import { asValue } from "awilix"
+import { loadInternalModule } from "./utils"
+
 import {
-  ClassConstructor,
-  ConfigModule,
-  LoaderOptions,
   Logger,
   MedusaContainer,
-  ModuleExports,
   ModuleResolution,
-  MODULE_RESOURCE_TYPE,
   MODULE_SCOPE,
-} from "../types/module"
+} from "../types"
 
 import { ModulesHelper } from "../module-helper"
 
 export const moduleHelper = new ModulesHelper()
 
-const registerModule = async (
+async function loadModule(
   container: MedusaContainer,
   resolution: ModuleResolution,
-  configModule: ConfigModule,
   logger: Logger
-): Promise<{ error?: Error } | void> => {
-  const constainerName = resolution.definition.registrationName
+): Promise<{ error?: Error } | void> {
+  const containerName = resolution.definition.registrationName
```

Comment:
suggestion: `containerName`renaming the constant to `registrationName`

Reply:
I had that changed in another PR, but will change it here as well